plugins/Diagnostics/Commands/ArchivingStatus.php

```diff
@@ -0,0 +1,106 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\Diagnostics\Commands;
+
+use Piwik\Container\StaticContainer;
+use Piwik\Mail;
+use Piwik\Plugin\ConsoleCommand;
+use Symfony\Component\Console\Output\BufferedOutput;
+use Symfony\Component\Console\Output\ConsoleOutput;
+use Symfony\Component\Console\Output\OutputInterface;
```

Comment:
We should avoid using Symfony components directly in our command files where possible. This might make prefixing them for wordpress more complex. If we need anything we should try to add wrapper methods to `ConsoleCommand` instead of using them directly.

Reply:
Good spot! I've added wrapper classes for these :+1: